docs/src/protocol_library.md

```diff
@@ -50,9 +50,9 @@ Account procedures can be used to read and write to account storage, add or remo
 | `get_vault_root` | Returns the vault root of the current account.<br><br>Inputs: `[]`<br>Outputs: `[VAULT_ROOT]` | Any |
 | `was_procedure_called` | Returns 1 if a procedure was called during transaction execution, and 0 otherwise.<br><br>Inputs: `[PROC_ROOT]`<br>Outputs: `[was_called]` | Any |
 
-## Note Procedures (`miden::note`)
+## Active Note Procedures (`miden::active_note`)
 
-Note procedures can be used to fetch data from the active note.
+Active note procedures can be used to fetch data from the active note.
```

Comment:
I think regardless of the name, I'd explain here again what "active note" means, or whatever other term we end up using. E.g.:

> Active note procedures an be used to fetch data from the note that is currently being processed by the transaction kernel.